tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -500,9 +498,9 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   if (gpu_enabled) {
     // TODO: Support cpu stitch with splat const
     pm.addNestedPass<FuncOp>(disc_ral::createDiscFuseSplatConstPass());
-    pm.addNestedPass<FuncOp>(
-        disc_ral::createDiscSpecializeFusionWithSpeculationPass(
-            gpu_options.sm_count, gpu_options.max_threads_per_sm));
+    // pm.addNestedPass<FuncOp>(
+    //     disc_ral::createDiscSpecializeFusionWithSpeculationPass(
```

Comment:
Any problems with it?

Reply:
just easier to analysis buffer living range, will remove this comment.